proto/stream_plan.proto

```diff
@@ -452,6 +455,9 @@ message ChainNode {
 
   // The rate limit for the chain node.
   optional uint32 rate_limit = 8;
+
+  // The state table used by ArrangementBackfill to replicate upstream mview's state table.
+  catalog.Table arrangement_table = 9;
```

Comment:
`TableDesc` is insufficient to construct a replicated `state_table`. So we need this aws well in the chain node definition.